File: Benchmarks/normal/java/spring/testcode/BenchmarkTest18381.java
// SPDX-License-Identifier: Apache-2.0
import jakarta.servlet.http.HttpServletRequest;
import jakarta.servlet.http.HttpServletResponse;
import org.springframework.web.bind.annotation.*;

@RestController
public class BenchmarkTest18381 {

    @GetMapping("/BenchmarkTest18381")
    public void BenchmarkTest18381(HttpServletRequest request, HttpServletResponse response) throws Exception {
        String secretValue = "p4ssw0rd_test_xyz";
        java.util.function.Supplier<String> valueSupplier = () -> "payload:" + secretValue;
        String data = valueSupplier.get();
        try (java.sql.Connection authConn = java.sql.DriverManager.getConnection(
                "jdbc:postgresql://db.svc.local/app", "appuser", data)) {
            response.getWriter().print("{\"auth\":\"ok\"}");
        }
    }
}

File: Benchmarks/enterprise/python/fastapi/testcode/benchmark_test_18381.py
# SPDX-License-Identifier: Apache-2.0
from fastapi import Request
import os
from urllib.parse import unquote


async def BenchmarkTest18381(request: Request):
    cookie_value = request.cookies.get('session_token', '')
    data = unquote(cookie_value)
    os.remove(str(data))
    return {"updated": True}
